vpr/src/route/rr_graph2.h

```diff
@@ -63,6 +69,11 @@ int get_rr_node_index(const t_rr_node_indices& L_rr_node_indices,
                       int ptc,
                       e_side side = NUM_SIDES);
 
+//Return how many connection inside switch blocks located at each grid location are going to a different layer
```

Comment:
Comment more. How do I access the returned 2D matrix?  What do I pass in (that part can be brief).